cmd/context/create.go

```diff
@@ -238,3 +244,12 @@ func (*ContextUse) initEnvVars() {
 		}
 	}
 }
+
+func isNewUrl(u string) bool {
```

Comment:
Where is this checking that the URL is new?

Reply:
This is more like a "is Url?" func? why the new ? should be different from some previous url?